### PR TITLE
verify listening addresses in ServerTest

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

File: zuul-core/src/test/java/com/netflix/zuul/netty/server/ServerTest.java [hunk 1]
```diff
@@ -78,11 +85,37 @@ public int acceptorCount() {
...
+            InetSocketAddress socketAddress = new InetSocketAddress("127.0.0.1", port);
+            sock = new Socket();
+            sock.setSoTimeout(100);
+            sock.connect(socketAddress, 100);
+            OutputStream out = sock.getOutputStream();
+            out.write("Hello".getBytes(StandardCharsets.UTF_8));
+            out.flush();
+            out.close();
+        } catch (IOException ex) {
+            throw new RuntimeException(ex);
```
Comment (argha-c) [inline zuul-core/src/test/java/com/netflix/zuul/netty/server/ServerTest.java:111]: `fail(msg)`

File: zuul-core/src/test/java/com/netflix/zuul/netty/server/ServerTest.java [hunk 2]
```diff
@@ -78,11 +85,37 @@ public int acceptorCount() {
...
+        } finally {
+            try {
+                sock.close();
+            }
+            catch (Exception ignored) {
```
Comment (argha-c) [inline zuul-core/src/test/java/com/netflix/zuul/netty/server/ServerTest.java:116]: nit: formatting `{}`

**Review by argha-c — APPROVED** [review]

File: zuul-core/src/test/java/com/netflix/zuul/netty/server/ServerTest.java (hunk 1 above)
Comment (sullis) [reply zuul-core/src/test/java/com/netflix/zuul/netty/server/ServerTest.java]: done

File: zuul-core/src/test/java/com/netflix/zuul/netty/server/ServerTest.java (hunk 2 above)
Comment (sullis) [reply zuul-core/src/test/java/com/netflix/zuul/netty/server/ServerTest.java]: done

**Comment by sullis** [comment]
can you merge this? @argha-c

**Pull request merged**